dbt/models/proximity/docs.md

```diff
@@ -84,6 +84,14 @@ Distance from each PIN to the nearest golf course.
 **Primary Key**: `pin10`, `year`
 {% enddocs %}
 
+# dist_pin_to_grocery_store
+
+{% docs table_dist_pin_to_grocery_store %}
+Distance from each PIN to the nearest grocery store.
+
+**Primary Key**: `pin10`, `year`
```

Comment:
**suggestion:** Let's add a quick sentence in here about which OSM tags were used to query the grocery stores.